code/lightna/magento-os-frontend/template/page/header/account.phtml

```diff
@@ -0,0 +1,48 @@
+<?php
+/** @var Lightna\Magento\Data\Session $session */
+$isLoggedIn = $session->user->groupId !== 0;
+$iconClass = $isLoggedIn ? 'user__filled' : 'user__outline';
+?>
+<div class="icon-dropdown dropdown">
+    <div class="header__icon">
+        <a href="/customer/account" class="icon <?= $iconClass ?> cursor-default"></a>
+        <span class="header__icon-desc"><?= phrase('My Account') ?></span>
+    </div>
+    <?php if ($isLoggedIn): ?>
+        <div class="dropdown__wrap">
```

Comment:
todo: split like in minicart; possibility to add more links/change entirely

Reply:
it's the only rework I found, I'll cerate ticket now from yout comment